client/app/private-database/configuration/private-database-configuration.controller.js

```diff
@@ -24,41 +24,68 @@ angular.module("App").controller(
 
         getConfigurationDetails () {
             this.loading = true;
-            this.privateDatabaseService
+            return this.privateDatabaseService
                 .getConfigurationDetails(this.productId)
                 .then((config) => {
-                    this.configurations = config.details;
-                    this.baseConfiguration = angular.copy(config.details);
+                    this.configurations = config.details.map((field) => this.convertAvailableValues(field));
                 })
-                .catch((err) => {
-                    this.alerter.alertFromSWS(this.$scope.tr("privateDatabase_configuration_error"), err, this.$scope.alerts);
+                .catch(() => {
+                    this.alerter.error(this.translator.tr("privateDatabase_configuration_error"), this.$scope.alerts.main);
```

Comment:
Logging error with $log.error(.. could be interesting for debugging and sentry

Reply:
Would also be a good idea to actually retrieve the error from the API

Reply:
I think ```$q.reject(...)``` is the prefered way of pushing error to Sentry in the web manager. That was not the point of this PR, but of course I can do that.